modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -111,20 +109,25 @@ $ oc apply -f httpbin-vs.yaml
 $ oc create namespace curl
 ----
 
-. Deploy the `curl` client by applying the curl.yaml file:
+. Deploy the `curl` client by applying the `curl.yaml` file:
 +
 [source,terminal]
 ----
-$ oc apply -n curl -f https://raw.githubusercontent.com/istio/istio/refs/heads/release-1.24/samples/curl/curl.yaml
+$ oc apply -n curl -f https://raw.githubusercontent.com/openshift-service-mesh/istio/refs/heads/master/samples/curl/curl.yaml
 ----
 
-. Using the `curl` client, send a request to the `/headers` endpoint of the `httpbin` application through the ingress gateway `Service`. Set the `Host` header of the request to `httpbin.example.com` to match the host that the {istio} `Gateway` and `VirtualService` resources specify.
+. Set a `CURL_POD` variable with the name of the curl pod:
 +
 [source,terminal]
 ----
-CURL_POD=$(oc get pods -n curl -l app=curl -o jsonpath='{.items[*].metadata.name}')
+$ CURL_POD=$(oc get pods -n curl -l app=curl -o jsonpath='{.items[*].metadata.name}')
+----
 
-oc exec $CURL_POD -n curl -- \
+. Using the `curl` client, send a request to the `/headers` endpoint of the `httpbin` application through the ingress gateway `Service` resource. Set the `Host` header of the request to `httpbin.example.com` to match the host that the {istio} `Gateway` and `VirtualService` resources specify.
++
+----
+[source,terminal]
```

Comment:
These lines need to be switched so that the source block isn't in the actual command syntax:
```suggestion
[source,terminal]
----
```